site/docs/latest/admin/http.md

```diff
@@ -0,0 +1,317 @@
+---
+title: BookKeeper http endpoints
+---
+
+This document is to introducing BookKeeper http endpoints that could be used for BookKeeper administration.
+If user want to use this feature, parameter "httpServerEnabled" need to be set to "true" in file conf/bk_server.conf.
+
+## All the endpoints
+
+Currently all the http endpoints could be divided into these 4 components:
+1. Heartbeat: heartbeat for a specific bookie.
+1. Config: doing the server configuration for a specific bookie.
+1. Ledger: http endpoints related to ledgers.
+1. Bookie: http endpoints related to bookies.
+1. AutoRecovery: http endpoints related to auto recovery.
+
+## Heartbeat
+
+### Endpoint: /heartbeat
+* Method: GET
+* Description: Get heartbeat status for a specific bookie
+* Response: 
+
+| Code   | Description |
+|:-------|:------------|
+|200 | Successful operation |
+
+## Config
+
+### Endpoint: /api/v1/config/server_config
```

Comment:
The format of the response should be included in each call.

Reply:
Thanks. the response is construct of code and body, and most of the command the response body not contains too much info. Will add the usefule respone body format.